101-documentdb-account-create-multi-region-account/metadata.json

```diff
@@ -1,7 +1,7 @@
 { 
-   "itemDisplayName": "Create a Multi-Region DocumentDB Database Account", 
-   "description": "This template creates a DocumentDB Database Account with two regions with a default consistency level of Session.", 
-   "summary": "Create a Multi-Region DocumentDB Database Account", 
+   "itemDisplayName": "Create a Multi-Region CosmosDB Database Account", 
+   "description": "This template creates a CosmosDB Database Account with two regions with a default consistency level of Session.", 
```

Comment:
Change to "This template creates an Azure Cosmos DB account with two regions..."